### PR TITLE
Create abu_reyan_ahmed.md

### PR DESCRIPTION
**Pull request by abureyanahmed**

### CODE REVIEW

File: _students/abu_reyan_ahmed.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Abu Reyan Ahmed
+role: PhD Student
+photo:
```
Comment (abureyanahmed) [inline _students/abu_reyan_ahmed.md]: headshots/Reyan.jpg

**Comment by cscheid** [comment]
Hey Abu, thanks for starting this.

I don't think you included the actual photo. Can you add your photo to the PR request? I also think you will need to edit the markdown file (currently the name of the image shows up in a comment, but not in the file itself.)

**Comment by abureyanahmed** [comment]
Sorry for that. I have updated.

**Comment by cscheid** [comment]
Looks good, thanks!

**Pull request merged**